src/module.js

```diff
@@ -41,15 +41,30 @@ class ModuleInterface {
 
   // returns Promise<string>
   source() {
+    // rk 2016-06-24:
+    // We should consider using lively.resource here. Unfortunately
+    // System.fetch (at least with the current systemjs release) will not work in
+    // all cases b/c modules once loaded by the loaded get cached and System.fetch
+    // returns "" in those cases
+
```

Comment:
cache invalidation: Not without removing the loaded module unfortunately (SystemJS looks in System.defined if the module is already loaded).
Resource per module: this is probably how we end up doing it.